eclipse-scout-core/src/tabbox/SimpleTab.js

```diff
@@ -311,27 +311,36 @@ export default class SimpleTab extends Widget {
   }
 
   _renderStatus() {
-    this._statusContainerUsageCounter -= (this._statusIconDivs.length === 0 ? 0 : 1);
+    this._statusContainerUsageCounter -= (this._statusIconSpans.length === 0 ? 0 : 1);
 
-    this._statusIconDivs.forEach($statusIcon => {
+    this._statusIconSpans.forEach($statusIcon => {
       $statusIcon.remove();
     });
-    this._statusIconDivs = [];
+    this._statusIconSpans = [];
 
     if (this.status) {
       this.status.asFlatList().forEach(status => {
-        if (!status || !status.iconId) {
+        if (!status || (!status.iconId && !status.message)) {
           return;
         }
-        let $statusIcon = this._getOrCreate$StatusContainer().appendIcon(status.iconId, 'status');
-        if (status.cssClass()) {
-          $statusIcon.addClass(status.cssClass());
+        if (status.iconId) {
+          let $statusIcon = this._getOrCreate$StatusContainer().appendIcon(status.iconId, 'status');
+          if (status.cssClass()) {
+            $statusIcon.addClass(status.cssClass());
+          }
+          this._statusIconSpans.push($statusIcon);
+        }
+        if (status.message) {
```

Comment:
This may "break" existing apps, if they already set a message. I think the message should be invisible by default (display: none) and only visible for the notification badge message. We can change this behavior with Scout 23 along with a migration note.

Reply:
Added display: none rule in SimpleTab.less 👍🏼